includes/class-support-overrides.php

```diff
@@ -95,6 +95,7 @@ public function nerdpress_override_rocket_delay_js_exclusions( $excluded_strings
 		$excluded_strings[] = 'gtagTracker'; // Monster Insights
 		$excluded_strings[] = "mediavine";
 		$excluded_strings[] = "adthrive";
+		$excluded_strings[] = "ads.min.js";
```

Comment:
I thought you had to escape periods.

Reply:
And this is exactly why we're doing code reviews!  (I think it would still work, but the period would be a wildcard, yeah?)  I'll fix.

Reply:
Any single character, yup. But not what we intended here. Up to you to change or not.

Reply:
already fixed.